### ISSUE TITLE
Clean up the repo?

### ISSUE THREAD

**Issue by karthik** [datacarpentry/lesson-template-ARCHIVED#18]
One challenge I've found with SWC is repo clutter that continues to propagate as each person clones/forks/imports a template repo. In this one, why do we have a workshop landing page?

![image](https://cloud.githubusercontent.com/assets/138494/9098018/1a3814fa-3b7c-11e5-9b37-09c890974bdd.png)

https://github.com/datacarpentry/lesson-template/blob/gh-pages/_layouts/workshop.html

Can this and unnecessary includes, headers, footers be removed entirely?

**Comment by tracykteal** [comment]
Yes, I think we should remove things we're not using. Maybe the original idea was that there would be one _layouts directory with all the different components. I don't think we want to do that though? So, in that case I think we should clean things up.

**Comment by tracykteal** [comment]
I think this might be a part of the conversation from https://github.com/datacarpentry/lesson-template/issues/11 where SWC is thinking of consolidating setup files. I think we're pretty happy with the current lesson-template and using the local files though? 

One thing I think we should change, referring to https://github.com/datacarpentry/lesson-template/issues/11 is if we want to include setup information in the lesson, which might be nice, we're propagating a lot setup instructions we don't need. Instead we should probably have an 'install page' or something like that and just link to those instructions. I can work on that in a separate PR.

**Comment by gvwilson** [comment]
The current SWC lesson template 
(https://github.com/swcarpentry/lesson-template) has only 
_layouts/page.html, while the workshop template 
(https://github.com/swcarpentry/workshop-template) has only 
workshop.html (for the landing page) and page.html (for any other pages 
people feel they want to create).  workshop-template still has all the 
setup instructions inline in index.html, rather than using include 
files, because our experience with the latter in 2013-14 was that people 
found it easier to edit an HTML page than to figure out how inclusion 
directives worked - I prefer the latter approach, but I'm not a typical 
instructor.
